consult.el

```diff
@@ -576,10 +576,37 @@ Turn ARG into a list, and for each element either:
 - split it if it a string.
 - eval it if it is an expression."
   (mapcan (lambda (x)
-            (if (stringp x)
-                (split-string-and-unquote x)
-              (ensure-list (eval x 'lexical))))
-          (ensure-list arg)))
+	    (if (stringp x)
+		(split-string-and-unquote x)
+	      (ensure-list (eval x 'lexical))))
+	  (ensure-list arg)))
```

Comment:
(Minor issue - can you try to avoid such whitespace changes please?)

Reply:
Added a .dir-locals.el but won't commit it.